topics/ipfs/index.md

```diff
@@ -0,0 +1,19 @@
+---
+aliases: InterPlanetary-FileSystem, InterPlanetary FileSystem, Inter-Planetary FileSystem, IPFS, ipfs, InterPlanetary File System
+created_by: Juan Benet 
+display_name: IPFS
+logo: ipfs.png
+released: May 23, 2014
+short_description: InterPlanetary File System (IPFS) is a protocol and network designed to create a content-addressable, peer-to-peer method of storing and sharing hypermedia in a distributed file system
+topic: IPFS
+url: https://ipfs.io/
+wikipedia_url: https://en.wikipedia.org/wiki/InterPlanetary_File_System
+---
+
+IPFS is a peer-to-peer distributed file system that seeks to connect all computing devices with the same system of files.
```

Comment:
Can you join lines 13-19? This body will be rendered as Markdown so every line break you have will come out as a `<br>` HTML tag.

Reply:
Have not used a markdown before that behaves that way. Fixed this now.

Reference: 
![image](https://user-images.githubusercontent.com/459764/34784981-b83d2290-f627-11e7-958d-b40750958cd6.png)

https://help.github.com/articles/basic-writing-and-formatting-syntax/#paragraphs-and-line-breaks

Reply:
Might be a quirk of how we're rendering it. Check out the screenshot in https://github.com/github/explore/pull/129 for how the body was rendering when it had line breaks, to see an example.

Reply:
Strange huh, are you not using GFM on Topics like on the rest of Github?